rinja/src/lib.rs

```diff
@@ -67,6 +67,8 @@ mod html;
 use std::{fmt, io};
 
 pub use rinja_derive::Template;
+#[doc(hidden)]
+pub use {core, std};
```

Comment:
You can refer to `core` and `std` by prepending them with `::` in macros. Like `::core::whatever`. If not, then add a comment to explain why we want it. Also, it's really not great because it's not rare for users to do `use rinja::*;`.

Reply:
The current code already does that, but even fully-qualified names like `::core` and `::std` can be shadowed. See the example in `testing`.

But yeah, I didn't recognize the problem that `use rinja::*` would import the symbol `core`, too, which might be undesirable. Python is smarter than rust in a way: Symbols with a leading underscore are omitted from glob imports. Would be nice if rust did that, too.

I guess I'll simply move the re-exports into the existing `#[doc(hidden)] mod helpers`. The name already exists, so it can't introduce a new problem.

Reply:
A new module containing it would be better indeed. :+1: